guides/v2.3/mrg/b2b/Company.md

```diff
@@ -40,7 +40,7 @@ The following directories contain extensible UI components:
 
 * `Magento\Company\view\adminhtml\ui_component` -  customer listing, invitation form, sales order grid
 
-* `Magento\Company\view\base\ui_component` - company listing, complany creation form, customer creation form
+* `Magento\Company\view\base\ui_component` - company listing, company creation form, customer creation form
```

Comment:
You should exempt guides/v2.3/mrg from the spellcheck.since the content is coming from the module READMEs.

Reply:
Good point. Any other exceptions like this, anyone?